tests/bits/parameter_handler_backslash_03.expect=diff.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::a, b, c
```

Comment:
because the `\` is put in the wrong place in file `parameter_handler_backslash_03.prm` and the output will contain just the `a`. same for test 04